### PR TITLE
feat: add timeline and dynamic variable change to video player

### PR DESCRIPTION
**Pull request by justusmattern27**

### CODE REVIEW

File: packages/player/src/styles.scss [hunk 1]
```diff
@@ -15,6 +15,127 @@ $states: ('initial' 'loading' 'ready' 'error');
...
 }
 
+.controls {
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between; /* Align button to the left and timeline to the right */
```
Comment (hkonsti) [inline packages/player/src/styles.scss:25]: We could also remove all the comments here IMO . The styles are very self-explanatory. If you want to leave in some that's ok too, just an idea.

**Review by hkonsti — APPROVED** [review]
This is super cool. Good work!

File: packages/player/src/styles.scss (hunk 1 above)
Comment (justusmattern27) [reply packages/player/src/styles.scss:25]: no this is all ChatGPT, I don't know CSS

File: packages/player/index.html [hunk 2]
```diff
@@ -14,8 +14,26 @@
   <body>
     <script type="module" src="/src/main.ts"></script>
     <revideo-player
+      id="myRevideoPlayer"
       src="/@id/__x00__virtual:template"
       style="aspect-ratio: 16 / 9"
+      variables='{"fill":"white"}'
     ></revideo-player>
+    <script>
+      document.addEventListener('DOMContentLoaded', function () {
```
[the remark this answers is not in the document]
Comment (justusmattern27) [reply packages/player/index.html:23]: this is just an example that you can run when you develop the player. index.html is not part of the actual npm package

**Pull request merged**